src/lib/constants.ts

```diff
@@ -0,0 +1,29 @@
+import { DuneClient } from "@cowprotocol/ts-dune-client";
+
+// ====================================================== //
+// ====================== CONSTANTS ===================== //
+// ====================================================== //
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+export const API_KEY = process.env.DUNE_API_KEY!;
+export const Dune = new DuneClient(API_KEY);
```

Comment:
this is a bit tricky, we should be either making API_KEY required, and asserting it, or making it optional and leaving dune: Dune | undefined. there are no hints when running the project as to whats required to resolve error if no env provided

Reply:
hmm yeah excellent point!